keyboards/cospad/keymaps/split_plus_and_zero/keymap.c

```diff
@@ -0,0 +1,81 @@
+#include QMK_KEYBOARD_H
+
+/* 
+ *  This keymap is based on the default keymap for the cospad.
+ *  It was adapted for the split Plus and split 0 layout focusing on the "00" key.
+ */
+
+enum custom_keycodes {
+  DBL_ZRO = SAFE_RANGE,
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case DBL_ZRO:
+      if (record->event.pressed) {
+        // when keycode DBL_ZRO is pressed
+        SEND_STRING("00");
+      } else {
+        // when keycode DBL_ZRO is released
+      }
+      break;
+
+  }
+  return true;
+};
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _BL,
+    _FL
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Keymap _BL: (Base Layer) Default Layer
+     * ,-------------------.
+     * |Esc |TAB | FN | BS |
+     * |----|----|----|----|
+     * | NL | /  | *  | ~  |
+     * |----|----|----|----|
+     * | 7  | 8  | 9  | -  |
```

Comment:
Tilde and minus are the wrong way around, with respect to the layout:
```suggestion
     * | NL | /  | *  | -  |
     * |----|----|----|----|
     * | 7  | 8  | 9  | ~  |
```